### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by v-silakshmi**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b9018ef1da8ac52ef591ed1565a7a46477afde2c/msteams-platform/messaging-extensions/how-to/add-authentication.md) | 85 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b4df08ce-2ebb-4e1d-a59a-5d357498c756) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/how-to/add-authentication.md [hunk 1]
```diff
@@ -36,13 +36,13 @@ The sequence is as follows:
 1. After the user signs in, you should close your window and send an **authentication code** to the Teams client.
 1. The Teams client then reissues the query to your service, which includes the authentication code passed in Step 5.
 
-Your service should verify that the authentication code received in step 6 matches the one from step 5. This ensures that a malicious user doesn't try to spoof or compromise the sign in flow. This effectively "closes the loop" to finish the secure authentication sequence.
```
Comment (v-habibaumme) [inline msteams-platform/messaging-extensions/how-to/add-authentication.md:39]: retain "that"

File: msteams-platform/messaging-extensions/how-to/add-authentication.md [hunk 2]
```diff
@@ -36,13 +36,13 @@ The sequence is as follows:
...
+Your service should verify the authentication code received in step 6 matches the one from step 5. Verification ensures, a malicious user doesn't try to spoof or compromise the sign in flow. This effectively "closes the loop" to finish the secure authentication sequence.
```
Comment (v-habibaumme) [inline msteams-platform/messaging-extensions/how-to/add-authentication.md:39]: Retain as is

File: msteams-platform/messaging-extensions/how-to/add-authentication.md [hunk 3]
```diff
@@ -74,7 +74,7 @@ As with other embedded experiences running inside Microsoft Teams, your code ins
 
 When the sign in request completes and redirects back to your page, it must perform the following steps:
 
-1. Generate a security code, a random number. You must cache this code on your service, along with the credentials obtained through the sign in flow, such as OAuth 2.0 tokens.
+1. Generate a security code, a random number. This code must be cached on your service, along with the credentials obtained through the sign-in flow, such as OAuth 2.0 tokens.
```
Comment (v-habibaumme) [inline msteams-platform/messaging-extensions/how-to/add-authentication.md:77]: Retain as is

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c0cf99d872e14e9a4f93c3ec1969fc0471ffcd28/msteams-platform/messaging-extensions/how-to/add-authentication.md) | 81 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ce962dab-e015-47d1-9f9c-4301d65164d0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/how-to/add-authentication.md (hunk 1 above)
Comment (v-silakshmi) [reply msteams-platform/messaging-extensions/how-to/add-authentication.md]: Correction is made

File: msteams-platform/messaging-extensions/how-to/add-authentication.md (hunk 2 above)
Comment (v-silakshmi) [reply msteams-platform/messaging-extensions/how-to/add-authentication.md]: correction is made

File: msteams-platform/messaging-extensions/how-to/add-authentication.md (hunk 3 above)
Comment (v-silakshmi) [reply msteams-platform/messaging-extensions/how-to/add-authentication.md]: correction is made

File: msteams-platform/messaging-extensions/how-to/add-authentication.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ Every request to your services includes the user ID, the user's display name, an
 },
 ```
 
-The `id` and `aadObjectId` values are guaranteed for the authenticated Teams user. They're used as keys to look up the credentials or any cached state in your service. In addition, each request contains the Azure Active Directory tenant ID, which is used to identify the user’s organization. If applicable, the request also contains the team ID and channel ID from which the request is originated.
```
Comment (surbhigupta12) [inline msteams-platform/messaging-extensions/how-to/add-authentication.md]: Why was the word "also" removed? Please revert

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/54b945bff5a8d18946b345287864676902a61a8c/msteams-platform/messaging-extensions/how-to/add-authentication.md) | 82 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4312d780-9b29-41b1-b35e-389d322eb03c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/54b945bff5a8d18946b345287864676902a61a8c/msteams-platform/messaging-extensions/how-to/add-authentication.md) | 82 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f63f3125-4c60-4f8c-b7a1-505826320969) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**